backend/src/main/java/asegroup1/api/controllers/LandRegistryController.java

```diff
@@ -77,6 +80,164 @@ public LandRegistryController(LandRegistryServiceImpl landRegistryService) {
         }
     }
 
+    @GetMapping(value = "get-display-data-test", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<String> getTestDisplayData() {
+        return new ResponseEntity<>("[\n" +
```

Comment:
Can you move this string out of the code, into a properties file, or a static string